iterations_runners/iterations_runner.java

```diff
@@ -131,7 +131,10 @@ public void getInstJSON(StringBuilder sb, int iterNum) {
         System.loadLibrary("kruntime");
     }
 
+    static long UNINITIALISED_TSR = -1;  // All ones
```

Comment:
Please see my other comment (in essence, -1 is an odd default, and suggests something clever is going on -- when it isn't).